languagetool-core/src/main/java/org/languagetool/Language.java

```diff
@@ -93,6 +100,12 @@ public abstract class Language {
   private Chunker postDisambiguationChunker;
   private Synthesizer synthesizer;
 
+  private final String shortCodeWithCountryAndVariant;
+
+  protected Language() {
```

Comment:
This is not used by -> shortCodeWithCountryAndVariant is always NULL

Reply:
this is impossible, the field is final and not initializing it (by adding another constructor) will cause a compiler error
and in subclasses, superclass constructors are always called, either explicit or implicit